src/main/java/lotto/view/ResultView.java

```diff
@@ -0,0 +1,29 @@
+package lotto.view;
+
+import lotto.domain.Lotto;
+
+import java.util.List;
+import java.util.Map;
+
+public class ResultView {
+    public static void printLottoCount(int lottoPrice) {
+        System.out.println(lottoPrice + "개를 구매했습니다.");
```

Comment:
String.format()을 사용해봐도 좋을거 같아요! 😄 
```suggestion
       System.out.println(String.format("%d 개를 구매했습니다.", lottoPrice));
```

Reply:
String.format()을 사용하는 방식으로 다 변경 후에 상수화도 진행했습니다! 감사합니다!